action_server/src/sema4ai/action_server/_new_project.py

```diff
@@ -83,3 +87,51 @@ def create_new_project(directory: str = ".", template_name: str = "") -> int:
         )
 
         return 1
+
+
+def list_templates(print_json: bool = False) -> int:
```

Comment:
Suggestion: rename to `_handle_list_templates` (as `list_templates` seems like it'll return the templates, instead of printing them).

Reply:
Yeah, I was considering that - however, it feels like `create_new_project` would fall a bit out of convention for this file. Do you reckon changing it to `handle_new_project` would make sense?

Reply:
I agree.

Reply:
Great! I think it will also align with Matti's changes nicely, as he used similar naming convention